examples/notebooks/tests/test_notebooks_e2e.py

```diff
@@ -86,7 +90,10 @@ def _delete_other_files(notebooks_dir: Path):
     for file in _OTHER_FILES:
         file_path = notebooks_dir / file
         if file_path.exists():
-            file_path.unlink()
+            if file_path.is_dir():
+                shutil.rmtree(file_path)
```

Comment:
I had avoided performing an `rmtree` and instead opted to explicitly list individual files to avoid accidentally deleting developer data